skel/share/defaults/gplazma.properties

```diff
@@ -519,6 +519,18 @@ gplazma.oidc.access-token-cache.expire.unit = SECONDS
 #
 gplazma.oidc.audience-targets =
 
+# ---- OpenID Connect Token Exchange plugin
+#
+
+gplazma.oidc-te.url = https://keycloak.desy.de/auth/realms/production/protocol/openid-connect/token
+gplazma.oidc-te.client-id = token-exchange 
+gplazma.oidc-te.client-secret = xxx
+gplazma.oidc-te.grant-type = urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Atoken-exchange
```

Comment:
this kind of format is not really user-friendly. You should take a regular string and encode it in the plugin.

Reply:
changed the parameters to a more user-friendly format.